### ISSUE TITLE
Pane resizing shortcut

### ISSUE THREAD

**Issue by OmriLevy888** [zeit/hyper#4576]
Using hyper, there is one feature I'm missing and couldn't for the life of me find it -> resizing panes using only the keyboard. I'm only Ubuntu 20.4, using hyper 3.0.2.

I came across hyerinator and hyperlayour which would allow me to create layouts with asymmetrical pane sizes but what I want is something like terminator's ctrl+shift+arrow keys to be able to resize a pane.

Does this exist/any plans on making this?

**Comment by bet4it** [comment]
It's really easy to be achieved by a plugin, but firstly, someone need to write this plugin😅

**Comment by OmriLevy888** [comment]
@bet4it yeah I assumed as much, guess it gives me a reason to learn some JS for a change, was just hoping maybe someone already came across it and made it
